client/ui/qml/Pages2/PageHome.qml

```diff
@@ -124,6 +124,9 @@ PageType {
                 text: ServersModel.defaultServerDescriptionCollapsed
             }
 
+            Component.onCompleted: {
```

Comment:
It shouldn't be called here.
Also required refactoring for the Android part in the SettingsController::SettingsController